src/wallet/wallet.cpp

```diff
@@ -3831,7 +3831,7 @@ std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(interfaces::Chain& chain,
     }
 
     if (gArgs.GetBoolArg("-upgradewallet", false)) {
-        if (!UpgradeWallet(walletInstance, gArgs.GetBoolArg("-upgradewallet", 0), error, warnings)) {
+        if (!UpgradeWallet(gArgs.GetBoolArg("-upgradewallet", 0), error, warnings)) {
```

Comment:
in commit 1e48796c99b63aa8fa8451ce7b0c20759ea43500

This is a bug and it needs to call walletInstance->UpgradeWallet, but I see the code is removed in the next commit anyway

Reply:
Luckily it doesn't compile:

```
wallet/wallet.cpp:3831:14: error: call to non-static member function without an object argument
        if (!UpgradeWallet(gArgs.GetBoolArg("-upgradewallet", 0), error, warnings)) {
             ^~~~~~~~~~~~~
1 error generated.